MaterialSkin/Controls/MaterialDrawer.cs

```diff
@@ -626,6 +630,27 @@ protected override void OnMouseMove(MouseEventArgs e)
             Cursor = Cursors.Arrow;
         }
 
+        protected override void OnMouseEnter(EventArgs e)
+        {
+            base.OnMouseEnter(e);
+            if (AutoShow && _isOpen==false)
+            {
+                Show();
+            }
+        }
+
+        protected override void OnMouseLeave(EventArgs e)
+        {
+            base.OnMouseLeave(e);
+            if (AutoShow)
+            {
+                if (DisplayRectangle.Contains(Control.MousePosition) == false)
```

Comment:
I don't think this check is necessary, does it work when you remove this?

Reply:
In theory it isn't necessary. But I noticed unexpected triggering of OnMouseLeave event.
When removed, I can see drawer opening and closing at the same time, and after a click event is done (everywhere in form) then all works fine.